libs/pageserver_api/src/config.rs

```diff
@@ -43,6 +43,21 @@ pub struct NodeMetadata {
     pub other: HashMap<String, serde_json::Value>,
 }
 
+/// PostHog integration config.
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct PostHogConfig {
```

Comment:
nit: It looks very like `PostHogClientConfig`. Can we unify and use `PostHogClientConfig` everywhere?